spec/linter-spec.js

```diff
@@ -38,6 +42,19 @@ describe('Js-YAML provider for Linter', () => {
     ),
   );
 
+  it('finds something wrong with wrong-node-kind.yaml', () =>
+    waitsForPromise(() =>
+      atom.workspace.open(wrongNodeKind).then((editor) => {
+        const messages = lint(editor);
+        expect(messages.length).toEqual(1);
+        expect(messages[0].type).toEqual('Error');
+        expect(messages[0].text).toEqual('unknown tag !<!epsilon>');
+        expect(messages[0].filePath).toMatch(/.+wrong-node-kind\.yaml$/);
```

Comment:
This should be `.toBe(wrongNodeKind)`. If the paths aren't matching perfectly then something is wrong.